osu.Game/Overlays/Comments/DrawableComment.cs

```diff
@@ -51,8 +55,16 @@ public class DrawableComment : CompositeDrawable
         private ShowMoreRepliesButton showMoreButton;
         private ShowRepliesButton showRepliesButton;
         private ChevronButton chevronButton;
+        private LinkFlowContainer actionsContainer;
+        private LoadingSpinner actionsLoading;
         private DeletedCommentsCounter deletedCommentsCounter;
 
+        [Resolved]
+        private IDialogOverlay dialogOverlay { get; set; }
```

Comment:
To ease things with tests (ie. allow the old tests to exist without this dependency being provided), you could probably make this an optional dependency and delete immediately if no dialog overlay is available to confirm.

```suggestion
        [Resolved(canBeNull: true)]
        private IDialogOverlay dialogOverlay { get; set; }
```

Reply:
![изображение](https://user-images.githubusercontent.com/53872073/194715499-ce325653-b854-479d-92b1-540e78653e4e.png)
What does this error exactly mean? Both types have nullability enabled, and the field is nullable too. I also saw this error a lot of times in other classes.

Reply:
It's weird that you're seeing that, have you run a `dotnet restore`?

Reply:
Just ran it on `osu.Game` and `osu.Desktop` and made Rider reanalyze all the files just in case. They are still here across all the project.
![изображение](https://user-images.githubusercontent.com/53872073/195078923-cd9f9a1d-bc55-4f08-87d5-0b29d9efe516.png)

In `DrawableComment` (my initial question) too. Nothing changed.
![изображение](https://user-images.githubusercontent.com/53872073/195078876-860e8292-ae32-4524-879c-1cd423fab249.png)